packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/components/PowerStateCellRenderer.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { TableCell } from 'src/modules/Providers/utils';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { VSphereVM } from '@kubev2v/types';
+import { Tooltip } from '@patternfly/react-core';
+import { OffIcon, PowerOffIcon, UnknownIcon } from '@patternfly/react-icons';
+
+import { VMCellProps } from './VMCellProps';
+
+export const PowerStateCellRenderer: React.FC<VMCellProps> = ({ data }) => {
+  const powerState = (data?.vm as VSphereVM)?.powerState;
+  const { t } = useForkliftTranslation();
+  const [icon, tooltipText] =
+    powerState === 'poweredOn'
+      ? [<PowerOffIcon color="green" key="on" />, t('Powered on')]
+      : powerState === 'poweredOff'
+      ? [<OffIcon color="red" key="off" />, t('Powered off')]
+      : [<UnknownIcon key="unknown" />, t('Unknown power state')];
+
+  return (
+    <TableCell>
+      <Tooltip content={tooltipText}>{icon}</Tooltip>
```

Comment:
Icon only may not be clear, we can add text, for example "<On Icon> On" and "<Off Icon> Off", WDYT ?